.ci/travis/install.sh

```diff
@@ -40,6 +40,8 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     esac
     pyenv rehash
     pyenv activate conan
+else
+    pip install -r conans/requirements_linux.txt
```

Comment:
So finally this requirements_linux is not necessary if moving "distro" to the main requirements.txt file?